docs/networking/diagnostics/inspecting-network-connections-with-netstat/index.md

```diff
@@ -77,42 +87,390 @@ unix  2      [ ACC ]     STREAM     LISTENING     24523    public/showq
+|--------|-------------|
+| Proto | The protocol of the socket (`unix`). |
+| RefCnt | The reference count, which is the number of attached processes connected via this socket. |
+| Flags | Any flags associated with the socket. This will most often display `ACC`, short for SO_ACCEPTON, which is shown for unconnected sockets whose processes are waiting for connection requests. |
+| Type | The type of the socket: datagram/connectionless (`SOCK_DGRAM`), stream/connection (`SOCK_STREAM`), raw (`SOCK_RAW`), reliably-delivered messages (`SOCK_RDM`), sequential packet (`SOCK_SEQPACKET`), or the obsolete `SOCK_PACKET`. |
+| State | The state of the socket: `FREE` for unallocated sockets, `LISTENING` for sockets listening for connections, `CONNECTING` for sockets that are about to be connected, `CONNECTED` for connected sockets, and `DISCONNECTING` for disconnecting sockets. If the state is empty, the socket is not connected. Sockets in the `LISTENING` state will not be displayed by default. |
+| I-Node | The filesystem [inode](https://en.wikipedia.org/wiki/Inode) of the socket. |
+| Path | The filesystem path of the socket. |
+
+## Command Line Options
+
+Some important and frequently-used command line options of `netstat` are as follows:
+
+| Option | Definition |
+|--------|------------|
+| `-v` | Shows verbose output. |
+| `-r` | Displays the kernel routing tables. |
+| `-e` | Displays extended information for network connections. |
+| `-i` | Displays a table of all network interfaces. When used with `-a`, the output also includes interfaces that are not up. |
+| `-s` | Displays summary statistics for each protocol. |
+| `-W` | Avoids truncating IP addresses and provides as much screen space as needed to display them. |
+| `-n` | Displays numerical (IP) addresses, instead of resolving them to hostnames. |
+| `-A` | Allows you to specify the protocol family. Valid values are `inet`, `inet6`, `unix`, `ipx`, `ax25`, `netrom`, `econet`, `ddp` and `bluetooth`. |
+| `-t` | Displays TCP data only. |
+| `-u` | Displays UDP data only. |
+| `-4` | Displays IPv4 connections only. |
+| `-6` | Displays IPv6 connections only. |
+| `-c` | Displays information continuously (every second). |
+| `-p` | Displays the process ID and the name of the program that owns the socket. It requires root privileges for this. |
+| `-o` | Displays timer information. |
+| `-a` | Shows both listening and non-listening network connections and unix sockets. |
+| `-l` | Displays listening network connections and unix sockets, which are not shown by default. |
+| `-C` | Displays routing information from the route cache. |
+| `-g` | Displays multicast group membership information for IPv4 and IPv6. |
+
+The rest of this guide will put the most important of these command line options at work in order to help you learn their usage. However, nothing can replace experimenting with `netstat` on your own.
+
+## Sockets/Network Connections
+
+### Include the LISTENING State
```

Comment:
Added two subsections to the start of the examples that tell the user about `-l` and `-a` by themselves, without other flags present. Doing this because they are used throughout the other examples, and I thought it beneficial to introduce them ahead of the more complicated options combinations